CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## 45 Release Notes
 
+### Improved input and output transfer performance on PAPI v2
+
+Previous versions of Cromwell required two PAPI v2 actions per input or output file or directory. For large numbers of 
+inputs or outputs this was very inefficient and could cause job creation requests to fail for being excessively large.
+Cromwell 45 now consolidates transfers into a single action each for all inputs and all outputs.
+In testing, total execution time for a call with 800 inputs improved from more than 70 minutes to less than 20 minutes.
+
+Transfer action consolidation also opens up possibilities for future enhancements to Cromwell to further improve
```

Comment:
Personal opinion: I don't think this kind of commentary belongs in the changelog

Reply:
ok, whoever demos this can save this for the audience